youtubedl/tests/test_basic.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+
+from .context import core
+
+# Run tests with python -m unittest tests/test_basic.py
+
+
+# These are the methods used as interface with the rest of the program.
+# To include a method in Video, insert the method name here.
+videoBaseMethods = frozenset(['allVideoStreams', 'download', 'videoId',
+        'videoThumbnail', 'videoTitle'])
+
+class TestVideo(unittest.TestCase):
+
+    def setUp(self):
+        self.video = core.Video(
+            'https://A.com')
+        self.methods = [m for m in dir(self.video) if not m.startswith('__')
+            and m not in self.video.__dict__.keys()]
+
+    def testVideoInit(self):
```

Comment:
Great stuff. I'd appreciate if you leave a docstring commenting what does every test do. Even if the function/class can be of help. Commenting is always appreciated.